components/contact/default_message.coffee

```diff
@@ -3,5 +3,9 @@ module.exports = (artwork, partner) ->
     'Hello, I am interested in placing a bid on this work. ' +
```

Comment:
This is for the stand-alone `/inquiry/:artwork_id` route (serving Microgravity proxy).